requre/cassette.py

```diff
@@ -596,6 +606,20 @@ def transform_hashable(keys: List) -> List:
                 output.append(item)
         return output
 
+    def _pretty_dict_output(self, input_dict: Optional[dict] = None, depth=0, sep="  "):
+        if not isinstance(input_dict, dict):
+            return
+        for key, value in input_dict.items():
+            yield sep * depth + key
+            yield from self._pretty_dict_output(value, depth + 1)
+
+    def _printable_dict_output(self, input_dict: dict):
+        return (
+            "Current internal object structure:"
+            + "\n"
+            + "\n".join(list(self._pretty_dict_output(self.storage_object)))
+        )
```

Comment:
I also though about that, but I just want to show root dictionaries, not all data, so json, yaml will do that as well, but I have to filter out all other data as well, so it helps me just little bit and I have still go recursively through whole tree, otherwise also https://docs.python.org/3/library/pprint.html do that.